tests/aux-date.beancount

```diff
@@ -1,8 +1,29 @@
 option "operating_currency" "EUR"
 
 
-2018-03-17 * "Test"
+2018-03-17 * "Test with transaction aux date"
   alt-date: 2018-03-16
   Assets:Test                        10.00 EUR
   Equity:Opening-Balance            -10.00 EUR
 
+2018-03-18 * "Test with posting aux date"
+  Assets:Test                        10.00 EUR
+    alt-date: 2018-03-13
```

Comment:
i'm fine with using "aux-date" as default name for this, but then shouldn't we standardize on it on the tests too? (there are several occurrences of this in the test cases)